include/glow/Graph/Graph.h

```diff
@@ -212,7 +215,24 @@ class Function final : public Named {
 
   ~Function();
 
+  /// Add placeholder for metadata such as profiling.
+  void addMetaDataPlaceholder(Placeholder *PH) {
+    metaDataPlaceholders_.push_back(PH);
+  }
+
+  /// Get list of metaData placeholders.
+  const std::list<Placeholder *> &getMetadata() const {
```

Comment:
nit: For consistency can you capitalize "Data" here -- and perhaps also append Placeholders (e.g. `getMetaDataPlaceholders()`)?

Reply:
IMO it should be `Metadata` everywhere since it's one word, but I agree it should be consistent!

Reply:
I'll make them all Metadata, and append Placeholders to getMetadata.